docs/dev-env.md

```diff
@@ -56,3 +56,16 @@ and set up in `nix develop` shell, it doesn't exist from team's perspective.
 To use a different shell for `nix develop`, try `nix develop -c zsh`. You can alias it if
 don't want to remember about it. That's the recommended way to use a different shell
 for `nix develop`.
+
+After running `nix develop` a env variable will be set indicating it was successful.
+
+```
+$ echo $IN_NIX_SHELL
+impure
+```
+
+Some enviroments indicate the curren status by prepending the name of the env in your prompt, such as python.
+
+For Nix, this can be accomplished by;
```

Comment:
`current`. Please check your spelling!

Reply:
A good part about typos in documentation is they encourage people to submit a first PR fixing them and break psychological barrier to entry. I know because all my Open Source projects have plenty of typos, and I get PRs fixing them all the time. :D

Reply:
good call, I'll start running them through ispell in the shell